meta-xt-domx/recipes-extended/xen/xen_git.bbappend

```diff
@@ -1,5 +1,7 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+
 require xen-source.inc
 
-do_configure:append() {
```

Comment:
The idea of the xt-common is that it should be compilable on any hardware platform that we support, and it should contain as generic functions as possible.

Any hardware-specific things, like `xt-defaults.cfg` added in this commit should be provided on the upper levels - xt-rpi5/xt-rcar or even in the product.

For example, if `xt-rpi5` layer has it's own `xen_git.bbappend` with it's own xt_defconfig, then `oe_runmake xt_defconfig` from this recipe will use config provided by upper level.

That's why we do not need to remove `do_configure:append()` here and do not have to add Reness-specific config inside `xt-common`. Instead, we should provide the required config in `xt-rcar` or `xt-rpi5` or `xt-prod-devl-xxx`.

Taking aside, that this commit has nothing to do with an upgrade to scarthgap.

Reply:
This is workarround and this workarround forces me do another workarround on this workarround. As I described in commit message I can't use config fragments with this recipe. If this commit out of scope this PR I will do another PR. If you don't like this solution I will offer another one. As I told in commit message - this recipe is not correct from Yokto's point of view.